docs/demonstration/schoology-e2e.ipynb

```diff
@@ -18,7 +18,7 @@
     "## Getting Started\n",
     "\n",
     "1. Confirm you have [poetry](https://python-poetry.org) installed (`poetry --version`).\n",
-    "1. Follow the [notebook instructions](README.md) to install dependencies used by this notebook.\n",
+    "1. Follow the [notebook instructions](../../src/schoology-extractor/README.md) to install dependencies used by this notebook.\n",
```

Comment:
The original instructions are for installing the notebook's dependencies, not Schoology Extractor's dependencies. So I don't think this change is correct.